api/controllers/UserController.js

```diff
@@ -475,20 +537,32 @@ module.exports = {
       request.payload.verificationExpiryEmail = false;
     }
     if (request.payload.old_password && request.payload.new_password) {
-      logger.warn('Updating user password', { request, security: true });
+      logger.warn(
```

Comment:
Is this warning or info?

Reply:
According to OICT, if I remember right, they want all sensitive operations to be logged with a relatively high level of logging, which is why if I remember right I used warn instead of info. I'm happy to use info instead but I'm not sure it's OICT compliant...

Reply:
OICT checkpoints relating to logging :

- All access control decisions can be logged and all failed decisions are logged.
- All input validation failures are logged.
- Security logging controls provide the ability to log both success and failure events that are identified as security-relevant.
- an indication that this is a security relevant event (if mixed with other logs);
- The security logging system includes a log analysis tool which allows the analyst to search for log events based on combinations of search criteria across all fields in the log record format supported by this system.

I think as long as the logs are marked with "security" the log "level" (info/warn/debug/error) isn't specified by OICT.

Reply:
Ok, changed to info in my last commit.